Dockerfile

```diff
@@ -17,8 +17,9 @@ FROM gcr.io/distroless/cc
 ARG getenvoy_binary=getenvoy
 COPY ${getenvoy_binary} /
 
+# TODO: delete this when we build in default version
 ARG reference
```

Comment:
this is effectively a build arg. We won't need this anymore when we choose latest version. I'll definitely do this and soon

Reply:
next PR will kill that image